tfjs-backend-webgpu/src/webgpu_util.ts

```diff
@@ -15,9 +15,7 @@
  * =============================================================================
  */
 import {DataType, util} from '@tensorflow/tfjs-core';
-
-import {MAX_COMPUTE_PER_DIMENSION_DISPATCH_SIZE} from './constants';
-
+import * as device_limits from './device_limits';
```

Comment:
I prefer you directly use `import maxComputeWorkgroupsPerDimension ...`

Reply:
Done.